src/components/Playground/Playground.jsx

```diff
@@ -7,7 +7,10 @@ import { getAuth, onAuthStateChanged } from 'firebase/auth';
 import { useEffect, useState } from 'react';
 import GetMapDirectionsButton from '../GetMapDirectionsButton/GetMapDirectionsButton.jsx';
 import 'react-html5-camera-photo/build/css/index.css';
-
+import { Image } from '@chakra-ui/react';
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/latest/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Image' is defined but never used.